plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -48,9 +48,18 @@ export const EntityPicker = ({
     formatEntityRefTitle(e, { defaultKind }),
   );
 
-  const onSelect = (_: any, value: string | null) => {
-    onChange(value || '');
-  };
+  const onSelect = useCallback(
+    (_: any, value: string | null) => {
+      onChange(value || '');
+    },
+    [onChange],
+  );
+
+  useEffect(() => {
+    if (entityRefs?.length === 1) {
+      onSelect('', entityRefs[0]);
```

Comment:
Can we just call `onChange` here instead of calling `onSelect`?

Reply:
good point!  , we can 👍